Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.yml

```diff
@@ -42,10 +43,10 @@ configuration:
 - additionalinfo: Whether to send or not automatic notifications to the configured notifications channel.
   display: Automatic Notifications
```

Comment:
dont we need to update the display and additional info as well?

Reply:
Updated